src/components/filters/FilterSection/index.jsx

```diff
@@ -18,6 +19,23 @@ const filterVisibleStatusObj = (optionList, inputText) => {
   return res;
 };
 
+// hasAnyValueSelected returns true if any values are selected in this filterStatus.
```

Comment:
i think this comment requires updating ?

Reply:
Thank you, it does!